common/searchattribute/defs.go

```diff
@@ -81,6 +81,24 @@ const (
 	// define a custom ScheduleId search attribute, in which case the query using the ScheduleId would operate just like
 	// any other custom search attribute.
 	ScheduleID = "ScheduleId"
+
+	// PausedInfo is a search attribute that stores the information about paused entities in the workflow.
+	// Format of a single paused entity: "<key>:<value>".
+	//  * <key> is something that can be used to identify the filtering condition
+	//  * <value> is the value of the corresponding filtering condition.
+	// examples:
+	//   - for paused activities, manual pause, we may have 2 <key>:<value> pairs:
+	//     * "Activity:MyCoolActivityType"
+	//     * "Reason:ManualActivityPause"
+	//     * or
+	//     * "Policy:<some policy id>"
+	//   - for paused workflows, we may have the following <key>:<value> pairs:
+	//     * "Workflow:WorkflowID"
+	//     * "Reason:ManualWorkflowPause"
+	//   - for paused taskqueues:
+	//     * "TaskQueue:TaskQueueName"
+	//     * "Reason:ManualTaskQueuePause"
+	TemporalPausedInfo = "TemporalPausedInfo"
```

Comment:
nit: Should this be `PauseInfo` and not `PausedInfo`? Seems like a better way of phrasing it.